### ISSUE TITLE
Unexpected error scratch building cornice

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/the-new-hotness#167]
See https://bugzilla.redhat.com/show_bug.cgi?id=1430178

**Comment by jeremycline** [comment]
Seems like the specfile's source URL 404'd
```
cmd:  spectool -g /var/tmp/thn-fqCRLU/python-cornice.spec
return code:  22
stdout:
Getting https://pypi.python.org/packages/source/c/cornice/cornice-2.4.0.tar.gz to ./cornice-2.4.0.tar.gz

stderr:
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
```

That URL still 404s. Maybe the specfile's URL needs updating?

**Comment by bowlofeggs** [comment]
Ah cool. It might be good for the-new-hotness to catch this specific error and just mention that the URL is no good. Maybe we can keep this open for error handling?

**Comment by jeremycline** [comment]
I actually fixed this already in https://github.com/fedora-infra/the-new-hotness/pull/161 - it's been released in 0.8.1 which is running in staging and seems to be working just fine.
